sdk/cosmos/azure-cosmos/src/main/java/module-info.java

```diff
@@ -24,13 +24,18 @@
     requires micrometer.core;
     //  This is only required by guava shaded libraries
     requires java.logging;
+    requires reactor.core;
+    requires org.reactivestreams;
+    requires com.fasterxml.jackson.databind;
+    requires org.slf4j;
```

Comment:
These are not required as `com.azure.core` module already requires them transitively.

Reply:
done